File: VegaIT.Timesheet.Persistence/Models/Employee.cs
namespace VegaIT.Timesheet.Persistence.Models;

public class Employee
    {

	public Guid EmployeeId { get; set; }

    public String Name { get; set; }

    public String Username { get; set; }

    public String Password { get; set; }

    public String Email { get; set; }

    public float HoursPerWeek { get; set; }

    public String Status { get; set; }

    public String Role { get; set; }


}

File: TimesheetFE/timesheet-app-fe/src/features/employees/models/Employee.ts
export class Employee {

    id: string
    name: string
    username: string
    password: string
    email: string
    hoursPerWeek: string
    status: string
    role: string

    constructor(id: string, name: string, username: string, password: string, email: string, hoursPerWeek: string, status: string, role: string) {
        this.id = id
        this.name = name
        this.username = username
        this.password = password
        this.email = email
        this.hoursPerWeek = hoursPerWeek
        this.status = status
        this.role = role
    }

}